tasks/Finals.tex

```diff
@@ -22,6 +22,20 @@ \subsection{Task}
 \begin{enumerate}[nosep]
 \item \textbf{Closing Furniture:} Doors of the Cabinet as well as the Dishwasher need to be closed.
 \item \textbf{Welcome Guest:} There is an additional Person waiting behind the Exit door.
+\item \textbf{Custom Tasks:} Additional reasonable household task.
+\end{enumerate}
+
+\subsubsection{Custom Tasks}
+\begin{enumerate}[nosep]
+\item \textbf{Requesting Additional Tasks:} Additional reasonable household tasks may be requested for scoring during the team leader meetings. These tasks must be approved by the \TC{}.
+
+\item \textbf{Arena-Specific Tasks:} Depending on the arena setup, certain household chores may be delegated to the robot for completion. Examples of tasks that may be handled by the robot include, but are not limited to, window cleaning and picture alignment on the wall.
+
+\item \textbf{Team-Supplied Items} In the event that additional items are necessary for the completion of a task (e.g., clothing for folding, watering cans), the requesting team is responsible for supplying these items.
+\begin{enumerate}[nosep]
+  \item \textbf{Standardization} These items must be regular household items (no markers, no custom printed handles etc.).
+  \item \textbf{Availability} All requested items must be provided and made available to the competitors prior to the day of the final. At least two identical copies of each item must be supplied to ensure adequate access task.
```

Comment:
Yes, this works for me. Just:
"must be supplied to ensure adequate access ~~task~~"